### DISCUSSION TITLE
How do I deploy Next.js app with pocketbase

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#2708] [Help]
### Summary

When I deploy my app with pocketbase,it show like this
![화면 캡처 2023-06-02 223709](https://github.com/vercel/community/assets/134067355/17fc6711-fbd8-432a-9c00-3c7479e79f74)
**How do I solve it**

### Steps to Reproduce

```tsx
async function getDocu(noteId: string) {
  const res = await fetch(
    `http://127.0.0.1:8090/api/collections/docu/records/${noteId}`,

    { cache: "no-store" }
  );
  const data = await res.json();
  return data;
}
export default async function docuPage({ params }: any) {
  const docu = await getDocu(params.id);
  return <h1>{docu?.title}</h1>;
}
```

**Comment by amyegan** [comment]
Do you see more error details in your browser's dev tools console?
